packages/widget-chat/src/actions/flags.js

```diff
@@ -0,0 +1,98 @@
+export const ADD_FLAG = `ADD_FLAG`;
+function actionAddFlag(activity) {
+  return {
+    type: ADD_FLAG,
+    activity
+  };
+}
+
+export const BEGIN_RECEIVE_FLAGS = `BEGIN_RECEIVE_FLAGS`;
```

Comment:
We're using different wording for similar actions (e.g. UPDATE_STATUS, RETRIEVE, etc). Can we decide on a set of naming schemes and use that? I'm partial to clumping all the status updates into a single action, but that may not be the best. What does everyone else think?